README.md

```diff
@@ -50,9 +50,11 @@ Memoization is easy to implement for code written in functional style; in many l
 
 See section [recursion](/recursion) for description.
 
-## Lamba Expressions
+## [Lambda Expressions](/lambda)
 
-*(Coming soon)*
+Lambda expressions, or anonymous functions, are not a concept unique to functional programming. However, functional programming uses lambda expressions in concepts.
+Lambda expressions are functions not bound to an identifier, a common example of this is in-line functions. In functional programming these are used
+in techniques such as currying, and high-order functions.
```

Comment:
(however, given the functional constraints, lambda expressions and true functions are indistinguishable)